oxd-server/src/test/java/org/gluu/oxd/server/mock/op/MockSetUpTest.java

```diff
@@ -0,0 +1,105 @@
+package org.gluu.oxd.server.mock.op;
+
+import com.google.common.base.Preconditions;
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.testing.DropwizardTestSupport;
+import io.dropwizard.testing.ResourceHelpers;
+import org.apache.commons.lang.StringUtils;
+import org.gluu.oxd.common.response.RegisterSiteResponse;
+import org.gluu.oxd.server.*;
+import org.gluu.oxd.server.persistence.PersistenceService;
+import org.gluu.oxd.server.service.RpService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
+
+/**
+ * Main class to set up and tear down suite.
+ *
+ * @author Yuriy Zabrovarnyy
+ * @version 0.9, 21/08/2013
+ */
+
+public class MockSetUpTest {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SetUpTest.class);
+
+    public static DropwizardTestSupport<OxdServerConfiguration> SUPPORT = null;
+
+
+    @Parameters({"host", "opHost", "redirectUrls"})
+    @BeforeSuite
+    public static void beforeSuite(String host, String opHost, String redirectUrls) {
+        try {
```

Comment:
All this is copy/paste from `org.gluu.oxd.server.SetUpTest#beforeSuite`. Lets refactor existing code and avoid it.